ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1796,6 +1797,14 @@ management_ssh:
   log_level: < SSH daemon log level >
 ```
 
+#### Management Tech-Support
+
+```yaml
+management_tech_support:
+  policy:
```

Comment:
To be closer to EOS command name and keep our style of list names being in plural, this key should be names something like `show_tech_support_policies`.
Alternatively we could consider a more elaborate data model like:
```
management_tech_support:
  policy_show_tech_support:
    exclude_commands:
      - command: < command_to_exclude >
        type: < text | json | default -> text >
    include_commands:
      - command: < command_to_include >
```

Reply:
Fixed